app/models/candidate_interface/contact_details_form.rb

```diff
@@ -0,0 +1,17 @@
+module CandidateInterface
+  class ContactDetailsForm
+    include ActiveModel::Model
+
+    attr_accessor :phone_number
+
+    def self.build_from_application(application_form)
+      new(phone_number: application_form.phone_number)
+    end
+
+    def save(application_form)
+      return false unless valid?
```

Comment:
maybe we do not need this line:
`If the object is invalid, the saving will fail and false will be returned.`
from https://apidock.com/rails/ActiveRecord/Persistence/update

Reply:
The YAGNI principle is the way for the spiritual purification 🌞

Reply:
Ah, I never heard of this before, makes sense. Will remove it.